opencga-analysis/src/main/java/org/opencb/opencga/analysis/cohort/CohortIndexTask.java

```diff
@@ -51,46 +35,46 @@ protected void check() throws Exception {
 
     @Override
     protected void run() throws Exception {
-        // Get all the studies
-        Query query = new Query();
-        QueryOptions options = new QueryOptions()
-                .append(QueryOptions.INCLUDE, Arrays.asList(StudyDBAdaptor.QueryParams.UID.key(),
-                        StudyDBAdaptor.QueryParams.ID.key(), StudyDBAdaptor.QueryParams.FQN.key(),
-                        StudyDBAdaptor.QueryParams.VARIABLE_SET.key()))
-                .append(DBAdaptor.INCLUDE_ACLS, true);
-        OpenCGAResult<Study> studyDataResult = catalogManager.getStudyManager().search(query, options, token);
-        if (studyDataResult.getNumResults() == 0) {
-            throw new CatalogException("Could not index catalog into solr. No studies found");
-        }
-
-        // Create solr collections if they don't exist
-        catalogSolrManager.createSolrCollections(CatalogSolrManager.COHORT_SOLR_COLLECTION);
-
-        for (Study study : studyDataResult.getResults()) {
-            Map<String, Set<String>> studyAcls = SolrConverterUtil
-                    .parseInternalOpenCGAAcls((List<Map<String, Object>>) study.getAttributes().get("OPENCGA_ACL"));
-            // We replace the current studyAcls for the parsed one
-            study.getAttributes().put("OPENCGA_ACL", studyAcls);
-
-            indexCohort(catalogSolrManager, study);
-        }
+//        // Get all the studies
```

Comment:
So.... have we removed this functionality? If so, remove it properly please :sweat_smile:

Reply:
Done